x-pack/plugins/security_solution/public/cases/components/user_action_tree/user_action_copy_link.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useCallback } from 'react';
+import { EuiToolTip, EuiButtonIcon } from '@elastic/eui';
+import { useParams } from 'react-router-dom';
+import copy from 'copy-to-clipboard';
+
+import { useFormatUrl } from '../../../common/components/link_to';
+import { SecurityPageName } from '../../../app/types';
+import * as i18n from './translations';
+
+interface UserActionCopyLinkProps {
+  id: string;
+}
+
+const UserActionCopyLinkComponent = ({ id }: UserActionCopyLinkProps) => {
+  const { detailName: caseId } = useParams<{ detailName: string }>();
+  const { formatUrl, search: urlSearch } = useFormatUrl(SecurityPageName.case);
+
+  const handleAnchorLink = useCallback(() => {
+    copy(formatUrl(`#${SecurityPageName.case}/${caseId}/${id}${urlSearch}`));
```

Comment:
```suggestion
    copy(formatUrl(getCaseDetailsUrl({ id: detailName })));
```
`formatUrl` should do everything for you. if not let me know.

Reply:
not sure this is working correctly, when i hit the copy button i dont get the root url. I get something like `/app/security/cases#case/80b4e670-f6a4-11ea-9a18-eb45ee28792a/9a4ea1f0-f6a6-11ea-9a18-eb45ee28792atimerange=(global:(linkTo:!(),timerange:(from:'2020-09-14T15:59:58.092Z',kind:absolute,to:'2020-09-14T16:00:58.092Z')),timeline:(linkTo:!(),timerange:(from:'2020-09-14T15:59:58.092Z',kind:absolute,to:'2020-09-14T16:00:58.092Z')))&timeline=(graphEventId:'',id:'6f413cf0-f6a3-11ea-9a18-eb45ee28792a',isOpen:!f)&?timerange=(global:(linkTo:!(),timerange:(from:'2020-09-14T15:59:58.092Z',kind:absolute,to:'2020-09-14T16:00:58.092Z')),timeline:(linkTo:!(),timerange:(from:'2020-09-14T15:59:58.092Z',kind:absolute,to:'2020-09-14T16:00:58.092Z')))&timeline=(graphEventId:'',id:'6f413cf0-f6a3-11ea-9a18-eb45ee28792a',isOpen:!f)`

Reply:
which that link, even if i do put it on the root url, redirects to all cases?!

Reply:
You are right! Fixed!